crypto/bindings/conversion.ts

```diff
@@ -10,9 +10,12 @@ import type {
 import type * as wasmNamespace from '../../compiled/node_bindings/plonk_wasm.cjs';
 import { bigIntToBytes, bytesToBigInt } from '../bigint-helpers.js';
 
+export { createRustConversion };
+
 type Field = Uint8Array;
 
-export { createRustConversion };
+// Kimchi_types.or_infinity
+type OrInfinity = MlOption<MlTuple<Field, Field>>;
```

Comment:
Nit: I would add an alias saying it is a point in affine form.

Reply:
Isn't it more helpful to be named like the OCaml type? I think that makes it easier to compare the OCaml and TS interfaces

Reply:
Ah! Yes, good point. It is copying `Kimchi_types.or_infinity`. It is better by being explicit then. However, to be fully correct, even though it is easy to make the conversion, it is not strictly equivalent as we have:
```
type nonrec 'f or_infinity = Infinity | Finite of ('f * 'f)
```
and not an option.

Reply:
oh that's true! the jsoo representation is the same -- but I can rewrite the type to be more clear

Reply:
The runtime representation is indeed the same :-)

Reply:
Yes, the runtime is the same. I would be in favour of the change for consistency and to stay in line with the native code, if it is fine for you.